circle.yml

```diff
@@ -6,6 +6,8 @@ stages:
     workDir: ~/go/src/github.com/palantir/godel
     machine:
       reusable: true
+      services:
+        - docker
```

Comment:
This is necessary but probably not quite sufficient -- I believe we have tests/integration tests that run in Docker containers, so those tests will probably not be able to run the Docker dist tests.

We may need to add logic to the tests that test for the existence of the `docker` command and then skip the tests if the command is not present -- presumably, running directly in CircleCI will be fine, but tests that run inside Docker containers will not be able to run the Docker dist tests.